mergify_engine/tests/unit/test_forwards.py

```diff
@@ -22,17 +22,10 @@
 import asyncmock
 from starlette import testclient
 
-from mergify_engine import tasks
 from mergify_engine import utils
 from mergify_engine import web
 
 
-# NOTE(sileht): Celery magic, this just skip amqp and execute tasks directly
-# So all REST API calls will block and execute celery tasks directly
-tasks.app.conf.task_always_eager = True
-tasks.app.conf.task_eager_propagates = True
-
-
```

Comment:
I've a doubt that this should be part of this PR?

Reply:
Good catch!

Reply:
Why I still see it there?

Reply:
Because I clean base.py that was having these lines too...